src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -621,8 +621,10 @@ AutoExifRotate =
 #    mkv      Matroska Video                                video/x-matroska
 #    mlp      Meridian Lossless Packing                     audio/vnd.dolby.mlp
 #    mov      QuickTime File Format                         video/quicktime
+#    mp2      MPEG Audio Layer II                           audio/mpeg
```

Comment:
@Nadahar Maybe we can add **(audio/vnd.dolby.dd-raw)** in the **ac3** line, as it could work better for some renderer and it's not easy to remember ?
> ORG_PN=PNG_LRG http-get:*:audio/vnd.dolby.dd-raw:DLNA.ORG_PN=AC3;DLNA.ORG_FLAGS=8D100000000000000000000000000000

For the same reason as above, **(video/vnd.dlna.mpeg-tts)** could be added in the **mpegts** line, as i tested and read that some renderers handle forward and rewind correctly with that one.

> http-get:*:video/vnd.dlna.mpeg-tts:DLNA.ORG_PN=MPEG_TS_HD_NA_XAC3_T;DLNA.ORG_FLAGS=ED100000000000000000000000000000
> http-get:*:video/vnd.dlna.mpeg-tts:DLNA.ORG_PN=MPEG_TS_SD_NA_XAC3;DLNA.ORG_FLAGS=ED100000000000000000000000000000
> http-get:*:video/vnd.dlna.mpeg-tts:DLNA.ORG_PN=MPEG_TS_MP_LL_AAC;DLNA.ORG_FLAGS=ED100000000000000000000000000000